platform/security/policy/security-policy-context/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -19,6 +19,11 @@
          name="Web Context Policy Manager"
          id="org.codice.ddf.security.policy.context.impl.PolicyManager">
 
+        <AD description="Depth to which paths will be traversed. Any value greater than 500 will be set to 500."
```

Comment:
Should we even support a path this deep? If they're not trying to DDOS the system, passing in some garbage like hundreds of slashes (which you're resolving) and were using a normal path with actual subcomponents in it, it'd be highly unlikely to contain 500 subelements.